pages/common/kinit.md

```diff
@@ -0,0 +1,29 @@
+# kinit
+
+> Authenticate a principal with a Kerberos server to gain and cache a ticket.
+> Note: A Kerberos principal can be either a user, service, or application.
+> More information: <https://web.mit.edu/kerberos/krb5-1.12/doc/user/user_commands/kinit.html>.
+
+- Authenticate user and obtain a ticket-granting ticket:
+
+`kinit {{username}}`
+
+- Renew a ticket-granting ticket:
+
+`kinit -R`
+
+- Specify a lifetime for the ticket:
+
+`kinit -l {{5h}}`
+
+- Specify a total renewable lifetime for the ticket:
+
+`kinit -r {{1w}}`
+
+- Specify a different principal name to authenticate as:
+
+`kinit -p {{principal@REALM}}`
+
+- Specify a different keytab file to authenticate with:
+
+`kinit -t {{/path/to/keytab}}`
```

Comment:
The path can be relative.
```suggestion
`kinit -t {{path/to/keytab}}`
```

Reply:
It can be, sure. But that doesn't mean that it should be in the example.

Reply:
interesting choice, thanks for linking that